receiver/mongodbatlasreceiver/alerts.go

```diff
@@ -182,30 +182,50 @@ func (a *alertsReceiver) startPolling(ctx context.Context, host component.Host)
 }
 
 func (a *alertsReceiver) retrieveAndProcessAlerts(ctx context.Context) error {
-	var alerts []mongodbatlas.Alert
+	numAlertsProcessed := 0
 	for _, p := range a.projects {
 		project, err := a.client.GetProject(ctx, p.Name)
 		if err != nil {
 			a.logger.Error("error retrieving project "+p.Name+":", zap.Error(err))
 			continue
 		}
-		projectAlerts, err := a.client.GetAlerts(ctx, project.ID, a.maxAlerts)
+		a.pollAndProcess(ctx, p, project, int64(numAlertsProcessed))
+	}
+	return a.writeCheckpoint(ctx)
+}
+
+func (a *alertsReceiver) pollAndProcess(ctx context.Context, pc *ProjectConfig, project *mongodbatlas.Project, numAlertsPolled int64) {
+	pageNum := 0
```

Comment:
Previously, this was initialized as `1`. Is `0` a valid page?

Reply:
It appears equivalent, was mostly copying what we did for organizations so I think we should keep it at 1. Nice catch

Reply:
If it is treated equivalently by the API, then we would presumably be getting the same page twice (once for 0, again for 1). So, agree w/ starting at 1.